src/main/java/com/aws/greengrass/provisioning/ProvisionConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.provisioning;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Are these redundant? Builders should be sufficient I think

Reply:
Since this is going to be used by plugin authors I am just exposing for convenience.

Reply:
I'd personally prefer keeping only what's needed since it keeps adding to the generated code